### PR TITLE
`CREATE LIKE` statements should preserve check-constraints on the new table.

### PR DESCRIPTION
**Pull request by nicktobey**

### CODE REVIEW

File: enginetest/queries/script_queries.go [hunk 1]
```diff
@@ -3918,6 +3918,23 @@ var CreateCheckConstraintsScripts = []ScriptTest{
 			},
 		},
 	},
+	{
```
Comment (fulghum) [inline enginetest/queries/script_queries.go:3921]: `create table ... like` tests probably make more sense to live with the other `create table` tests. Check out `TestCreateTable` and `queries.CreateTableQueries` and see if you can find a nice spot to nestle this into.

**Review by fulghum — APPROVED** [review]
Code change looks great; just a minor suggestion on where to put the test.

File: enginetest/queries/script_queries.go (hunk 1 above)
Comment (nicktobey) [reply enginetest/queries/script_queries.go]: Okay. I moved the test into create_table_queries.go. How's this?

Comment (fulghum) [reply enginetest/queries/script_queries.go]: Looks great! We have soooo many tests, it's helpful to try and keep them as organized as we can so they don't sprawl too much. 

Ship it! 🚢

**Pull request merged**